dts/bindings/display/nxp,imx-mipi-dsi.yaml

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright (c) 2022, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+description: NXP MCUX MIPI DSI
+
+compatible: "nxp,imx-mipi-dsi"
+
+include: mipi-dsi-host.yaml
+
+properties:
+    interrupts:
+      required: true
+
+    nxp,lcdif:
+      type: phandle
+      required: true
+      description:
+        Instance of the LCDIF peripheral.
+
+    dpi-color-coding:
+      type: string
+      enum:
+        - "16-bit config 1"
+        - "16-bit config 2"
+        - "16-bit config 3"
+        - "18-bit config 1"
+        - "18-bit config 2"
+        - "24-bit"
+      description:
+        MIPI DPI interface color coding.
+
+    dpi-pixel-packet:
+      type: string
+      enum:
+        - "16-bit"
+        - "18-bit"
+        - "18-bit loose"
+        - "24-bit"
+      description:
+        MIPI DSI pixel packet type send through DPI interface.
+
+    dpi-video-mode:
+      type: string
+      enum:
+        - "Non burst synch pulse"
+        - "Non burst synch event"
+        - "Burst"
+      description:
+        DPI video mode.
+
+    dpi-bllp-mode:
+      type: string
+      enum:
+        - "Low power"
+        - "Blank"
+        - "Null"
+      description:
+        Behavior in BLLP (Blanking or Low-Power Interval).
+
+    autoinsert-eotp:
+      type: boolean
+      required: true
```

Comment:
`type: boolean` with `required: true` is a little weird... are you sure? I think this should be dropped.

Reply:
Thank you @mbolivar-nordic. I have removed the line and have also moved the bindings file to the new `mipi-dsi` folder instead of `display`